### ISSUE TITLE
Add aria-hidden and role attribute to svg icons

### ISSUE THREAD

**Issue by clemlatz** [ynnoj/ember-cli-notifications#329]
`aria-hidden` et `role` attributes are required on decorative images for accessibility but are missing on notification icons.

I see this has been already addressed in commit https://github.com/mansona/ember-cli-notifications/commit/d2e5c51ca60d4fd07de2ab89d335ae67aca0d0f7 which is not included in the latest release.

Do you plan to publish a new release including this commit any time soon?

Thanks!
